### PR TITLE
Development

### PR DESCRIPTION
**Pull request by W3-Tariqul**
## Description

Added new user onboard process to start in Telemesh. Also changed user registration system. Added new Last name input section. And removed static image by user name.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

-The design flow was updated and fixed unexpected exceptions in the broadcast section

- [x] New feature (non-breaking change which adds functionality)
- Added new onboard screen
- Changed static image and it is replaced by the user's first and last name.
- Wallet backup process is added where users can backup their wallets.

## How Has This Been Tested?

- [x] Test A
Manually tested with new viper and SDK.  New onboarding process adjusted with service.APP. And also tested in local UI test
- [x] Test B

Tested the whole process in Travis-CI and it is successfully passed. And code coverage is 77%

**Test Configuration**:
* Firmware version: Android 
* Hardware: Android 11, Android 5.1.1
* Toolchain:
* SDK:  Android SDK 30

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/src/androidTest/java/com/w3engineers/unicef/telemesh/ui/createuser/CreateUserViewModelTest.java [hunk 1]
```diff
@@ -74,8 +74,9 @@ public void isNameValid_true_setValidString() {
     @Test
     public void storeData_true_setValidData() {
         String firstName = "Daniel";
+        String lastName = "Craig";
         String password = "mesh_123";
```
Comment (W3-Anjan) [inline app/src/androidTest/java/com/w3engineers/unicef/telemesh/ui/createuser/CreateUserViewModelTest.java:78]: is there any security concern on static password set here @W3-Tariqul?

Comment (W3-Tariqul) [reply app/src/androidTest/java/com/w3engineers/unicef/telemesh/ui/createuser/CreateUserViewModelTest.java:78]: No. It is the test account and password, and it is for Unit test.

**Review by W3-Anjan — APPROVED** [review]
We have made new onboarding and changed the previous app flow with the latest. It will satisfy the LTA TOR milestone too.

**Review by W3-Aziz — APPROVED** [review]
Added new onboarding process with these changes.
Changes are looking good. There is no sophisticated credential-related information found.
We can move the changes to the master branch.

**Pull request merged**